doc/dox_comments/header_files/ssl.h

```diff
@@ -2698,6 +2698,72 @@ WOLFSSL_API void wolfSSL_load_error_strings(void);
 */
 WOLFSSL_API int  wolfSSL_library_init(void);
 
+/*!
+    \brief This function sets the Device Id.
+
+    \return WOLFSSL_SUCCESS upon success.
+    \return BAD_FUNC_ARG if ssl is NULL.
+
+    \param ssl pointer to a SSL object, created with wolfSSL_new().
+
+    _Example_
+    \code
+    WOLFSSL* ssl;
+    int DevId = -2;
+
+    wolfSSL_SetDevId(ssl, devId);
+
+    \endcode
+
+    \sa wolfSSL_CTX_SetDevId
+    \sa wolfSSL_CTX_GetDevId
+*/
+WOLFSSL_API int wolfSSL_SetDevId(WOLFSSL* ssl, int devId)
+
+/*!
+    \brief This function sets the Device Id.
```

Comment:
Let's also add that this sets the device ID at the WOLFSSL_CTX context level, as opposed to the wolfSSL_SetDevId() that works at the WOLFSSL session level.  Also, I don't see a "\param" for the devId parameter.